module/User/src/Permissions/Assertion/IsBoardMember.php

```diff
@@ -36,25 +34,6 @@ public function assert(Acl $acl, RoleInterface $role = null, ResourceInterface $
         }
         $member = $role->getMember();
 
-        foreach ($member->getBoardInstallations() as $boardInstall) {
-            if ($this->isCurrentBoard($boardInstall)) {
-                return true;
-            }
-        }
-
-        return false;
-    }
-
-    /**
-     * Check if this is a current board member.
-     *
-     * @return bool
-     */
-    protected function isCurrentBoard(BoardMember $boardMember)
-    {
-        $now = new DateTime();
-
-        return $boardMember->getInstallDate() <= $now &&
-            (null === $boardMember->getDischargeDate() || $boardMember->getDischargeDate() >= $now);
+        return $member->isBoardMember();
     }
 }
```

Comment:
Something I forgot to mention, this assertion is not used anywhere else in the website and can finally be removed.

Reply:
I removed the assertion